index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-answer-screen">0</div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-AC">AC</button>
```

Comment:
For these specific button names, something like 'calculator-button__AC' would be a clearer naming convention. (The specific button type modifies the base calculator-button type and would never be used on something other than a calculator-button, so the calculator-button class should be part of the name)